src/main/java/frc/robot/Robot.java

```diff
@@ -8,6 +8,7 @@
 package frc.robot;
 
 import edu.wpi.first.wpilibj.TimedRobot;
+import edu.wpi.first.wpilibj.shuffleboard.Shuffleboard;
```

Comment:
Code Hygiene: Is this required in Robot.java? If not lets remove.

Reply:
well it needs to be called periodically